pyproject.toml

```
@@ -658,7 +658,7 @@ filterwarnings = [
 ]
 
 [tool.ruff]
-required-version = ">=0.3.4"
+required-version = "==0.3.7"
```

```suggestion
required-version = ">=0.3.7"
```
I would suggest using a lower bound here as before. That will make testing future releases locally easier.